packages/engineer/src/application-proxy-service.ts

```diff
@@ -86,7 +81,8 @@ export class TargetApplication extends Application {
         return super.getFeatureEnvDefinitions(features, configurations);
     }
 
-    public setNodeEnvManager(nem: NodeEnvironmentsManager) {
+    public setNodeEnvManager(nem: NodeEnvironmentsManager, nodeEnvironmentsMode?: LaunchEnvironmentMode) {
```

Comment:
no need for that.
When you do 'new NodeEnvironmentManager(..)' you set up the mode. any specific reason for saving the mode explicitly?

Reply:
You pass the mode on the request to runServerEnvironments, not to the node environment manager constructor

Reply:
Cool, my bad